src/component/admin/auth/AdminSignUp.js

```diff
@@ -0,0 +1,124 @@
+import {Box, Button, FormControl, Input, Link, Text, VStack} from "@chakra-ui/react";
+import {useState} from "react";
+import GeneralButton from "../../reusables/GeneralButton";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faEye } from "@fortawesome/free-solid-svg-icons";
+import axios from "axios";
+const eye = <FontAwesomeIcon bg={"white"} h={'1.7vh'} w={'2vh'} icon={faEye} />;
+
+
+const AdminSignUp = () => {
+    const[account, setAccount] = useState('');
+    const[email, setEmail] = useState('');
+    const[phone, setPhone] = useState('');
+    const[password, setPassword] = useState('');
+
+
+
+    const [passwordShown, setPasswordShown] = useState(false);
+
+    const togglePasswordVisibility = () => {
+        setPasswordShown(!passwordShown);
+    };
+
+    const onSubmit = (e) => {
+        e.preventDefault();
+        // localStorage.setItem('')
+        axios.post('http://localhost:8080/api/customer', this.state)
+            .then(response => {console.log(response)})
+        console.log("email  log " + email, "phone log " + phone, "password log  " + password)
```

Comment:
Same as above,
1. Handle errors
2. Show notifications to inform user of result of network call
3. Remove hard coded url